Source/Utils/WarpXUtil.cpp

```diff
@@ -213,3 +213,15 @@ WarpXParser makeParser (std::string const& parse_function, std::vector<std::stri
     }
     return parser;
 }
+
+namespace WarpXUtilMsg{
+
+void AlwaysAssert(int expression, const std::string& msg = "ERROR!")
+{
+    if(expression != 0) return;
```

Comment:
```suggestion
    if(expression == true) return;
```

Reply:
I've implemented a slightly different solution, inspired by your comment.
I've renamed `expression` (now a `bool`) as `is_expression_true`, so that I can write:  `if(is_expression_true) return;`